v0/destinations/snapchat_conversion/util.js

```diff
@@ -97,12 +106,53 @@ function getPriceSum(message) {
   return String(priceSum);
 }
 
+/**
+ * Create Mailchimp Batch payload based on the passed events
+ * @param {*} events
+ * @returns
+ */
+const generateBatchedPayloadForArray = events => {
```

Comment:
We have a mixture of ES5 functions and ES6 arrow functions syntax

Reply:
@sandeepdsvs please reply and close the comment

Reply:
Done